src/vivliostyle/table.js

```diff
@@ -1705,6 +1705,9 @@ goog.scope(function() {
      */
     vivliostyle.table.TableRowLayoutConstraint = function(nodeContext) {
         RepetitiveElementsOwnerLayoutConstraint.call(this, nodeContext);
+
+        /** @type {Array.<{constraints: Array.<adapt.layout.FragmentLayoutConstraint>, breakPosition:adapt.vtree.NodeContext}>} */
+        var cellFragmentConstraints = [];
```

Comment:
It seems that the field used in other methods is `this.cellFragmentLayoutConstraints`, not `this.cellFragmentConstraints`.